### PR TITLE
Selected ui highlighter

### PR DESCRIPTION
**Pull request by jb-perrier**
Very basic ui highlighter in order to fix this issue #70 : 

 demo

This PR need improvement, please post your suggestions below.

### CODE REVIEW

File: FlaxEditor/Windows/GameWindow.cs [hunk 1]
```diff
@@ -287,6 +289,18 @@ public override void Draw()
...
             }
+
+            var selection = Editor.Instance.SceneEditing.Selection;
+                
+            selection.ForEach((node) =>
+            {
+                if (node.EditableObject is UIControl controlActor)
+                {
+                    if (controlActor.Control != null)
+                        Render2D.DrawRectangle(controlActor.Control.Bounds, Color.Green, 5f);
```
Comment (stefnotch) [inline FlaxEditor/Windows/GameWindow.cs]: I suppose the 5f could be reduced to 1f to be more consistent with the existing selections in Flax

Comment (JimiVacarians) [reply FlaxEditor/Windows/GameWindow.cs]: It's nearly invisible with 1F.

Comment (stefnotch) [reply FlaxEditor/Windows/GameWindow.cs]: Ok, that makes sense. Does 2f look fine?

By the way, if you want to use the user set selection outline color, you should be able to access it with `Editor.Options.Visual.SelectionOutlineColor0`

File: FlaxEditor/Windows/GameWindow.cs [hunk 2]
```diff
@@ -287,6 +289,16 @@ public override void Draw()
                     Render2D.DrawRectangle(new Rectangle(new Vector2(4), Size - 8), Color.Orange * alpha);
                 }
             }
+            
+            Editor.Instance.SceneEditing.Selection.ForEach((node) =>
+            {
+                if (node.EditableObject is UIControl controlActor)
+                {
+                    if (controlActor.Control != null)
+                        Render2D.DrawRectangle(controlActor.Control.Bounds, Editor.Instance.Options.Options.Visual.SelectionOutlineColor0, 2f);
```
Comment (stefnotch) [inline FlaxEditor/Windows/GameWindow.cs]: Hold up, is it just me or are you not using the `UISelectionOutlineSize` here?

Comment (JimiVacarians) [reply FlaxEditor/Windows/GameWindow.cs]: Omg, fixed.

**Comment by jb-perrier** [comment]
This feature is very primitive looking ( a raw border ), do i need to improve it or just close it ?

**Comment by mafiesto4** [comment]
Thanks for your contribution! We will continue improving UI tools during the next updates.
Here is a tracking ticket for this area: https://trello.com/c/t6kezmsS/322-improve-ui-workflow

**Pull request merged**